jvb/pom.xml

```diff
@@ -17,6 +17,7 @@
   <url>https://github.com/jitsi/jitsi-videobridge</url>
   <properties>
     <assembly.skipAssembly>false</assembly.skipAssembly>
+    <kotest.version>4.1.2</kotest.version>
```

Comment:
Any reason not to use 4.1.3?

Reply:
I looked [here](https://github.com/kotest/kotest) and saw latest release as 4.1.2 so used that...I'll change to 4.1.3